js/bibliographicCitationView.js

```diff
@@ -305,7 +305,7 @@ class BibliographicCitationView extends HTMLElement {
 
     let hash = window.location.hash
     if (hash.startsWith("#")) {
-      hash = window.location.hash.substring(1)
+      hash = window.location.hash?.slice(1)
```

Comment:
This was not code introduce by this PR.  It was already accepted and approved through OSS.  I just changed the use of substring() since I was doing it everywhere else.